packages/core/src/error.ts

```diff
@@ -50,6 +50,7 @@ export enum ErrorCode {
 
     NOT_FOUND = "not_found",
     INVALID_CSV = "invalid_csv",
+    INVALID_1PUX = "invalid_1pux",
```

Comment:
I didn't see any translation/text for this. Not sure if I missed something.

Reply:
The error codes are kind of arbitrary and not really used consistently. I'm actually thinking about using more generic error codes in general. `ErrorCode.ENCODING_ERROR` would do here as well for example.

Reply:
Ok. I'll keep it. Let me know if you'd prefer I replace it with `ENCODING_ERROR`.